assembled:
  differential_drive:
    wheel_separation: 0.18
    wheel_radius: 0.04
    wheel_max_speed: 500.0
    wheel_min_speed: 200.0
  